.gitignore

```diff
@@ -12,3 +12,8 @@ tox.ini
 /.vscode
 .eggs
 *.code-workspace
+
+# Rhino Backup files
+*.3dm
```

Comment:
We probably want to keep track of changes in 3dm files. Otherwise, the modifications to Rhino files for testing will be ignored.